app/models/reports/export_raw.rb

```diff
@@ -84,6 +84,8 @@ def default_report_hash
         journal_date: ->(od) { od.journal.journal_date if od.journal },
         reconciled_note: :reconciled_note,
         reconciled_at: :reconciled_at,
+        invoice_number: ->(od) { od.statement.try(:invoice_number) },
+        charge_for: ->(od) { od.price_policy.try(:charge_for) },
```

Comment:
This might be slightly more complicated than we initially hoped.

We might need to talk through this with the client. Here are the scenarios I can think of:

* Price policy is present, but does not respond to charge_for because it's an Item or ServicePricePolicy.
  * In that case, we'll want to display "Quantity".
* Price policy is blank because the order hasn't been completed
  * Maybe we should display the current price policy's charge_for. Or do we leave it blank?
* Price policy is blank because it's a problem & missing actuals
  * We would probably want to do the same as we do for being uncomplete
* Price policy is blank because it's a problem & missing price policy
  * Probably leave it blank in this case.